packages/empty-state/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@crave/farmblocks-empty-state",
+  "version": "0.1.0",
+  "description": "Farmblocks empty state",
+  "author": "Crave Food Services and AUTHORS",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "files": ["AUTHORS", "LICENSE", "README.md", "CHANGELOG.md", "lib"],
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "clean": "rm -rf lib",
+    "build": "yarn clean && babel src -d lib --ignore '*.test.js,*.story.js'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CraveFood/farmblocks.git"
+  },
+  "bugs": {
+    "url": "https://github.com/CraveFood/farmblocks/issues"
+  },
+  "homepage": "https://github.com/CraveFood/farmblocks#readme",
+  "keywords": [
+    "farmblocks",
+    "react",
+    "component",
+    "view",
+    "empty state",
+    "placeholder",
+    "call to action"
+  ],
+  "peerDependencies": {
+    "prop-types": "^15.6.0",
+    "react": "^16.0.0",
+    "styled-components": "^2.2.1"
+  },
+  "dependencies": {
+    "@crave/farmblocks-button": "^1.0.7",
+    "@crave/farmblocks-image": "^0.2.1",
+    "@crave/farmblocks-text": "^1.1.1"
```

Comment:
Update `@crave/farmblocks-text` to use the latest release that fixes the `title` div property warning